data-processor/src/main/java/io/micronaut/data/processor/visitors/RepositoryTypeElementVisitor.java

```diff
@@ -321,6 +322,11 @@ public void visitMethod(MethodElement element, VisitorContext context) {
                 context.fail(matchContext.getUnableToImplementMessage() + e.getMessage(), e.getElement() == null ? element : e.getElement());
                 this.failing = true;
             } catch (Exception e) {
+                if (e instanceof ElementPostponedToNextRoundException || e.getClass().getSimpleName().equals("PostponeToNextRoundException")) {
```

Comment:
Do we need this in 4.12.x (with core 4.8.7)?

Reply:
yes